File: Easy/Python 3/Pascal's_Triangle(118).py
class Solution:
    def generate(self, numRows: int) -> List[List[int]]:
        l=[[]]
        for i in range(numRows):
            c=[]
            for j in range(i+1):
                n=math.factorial(i)//(math.factorial(i-j)*math.factorial(j))
                c.append(n)
            l.append(c)
        return l[1:]

File: Easy/C++/Pascal's_Triangle(118).cpp
class Solution {
public:
    vector<vector<int>> generate(int numRows) {
        vector<vector<int>> res;
        for(int i=0;i<numRows;i++)
        {
            vector<int> r1(i+1,1);
            res.push_back(r1);
            if(i>=2)
            {
                for(int j=1;j<i;j++)
                {
                    res[i][j]=res[i-1][j-1]+res[i-1][j];
                }
            }
        }
        return res;
    }
};